packages/react/src/components/form/Field/FieldCounter.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useRef, useState } from 'react';
+import { ValidationMessage } from '../../ValidationMessage';
+
+type CharacterCounterProps = {
+  /** Label template for when `maxCount` is exceeded
+   * @default '%d tegn for mye'
+   */
+  over?: string;
+  /** Label template for count
+   * @default '%d tegn igjen'
+   */
+  under?: string;
+  /** The maximum allowed characters. */
+  maxCount: number;
```

Comment:
Could this just be `limit` or `max`  since `Counter` is already part of the component name? ☺️

Reply:
Hmm, maybe `limit`?

Reply:
I'll fix.